packages/fluentui/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -859,12 +891,24 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     getInputProps: (options?: GetInputPropsOptions) => any
   ) => (predefinedProps: DropdownSearchInputProps) => {
     const handleInputBlur = (e: React.SyntheticEvent, searchInputProps: DropdownSearchInputProps) => {
+      if (disabled) {
+        e.preventDefault();
```

Comment:
Also not sure about this as `disabled` input is not focusable:

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_input_disabled